src/components/descriptions.jsx

```diff
@@ -0,0 +1,57 @@
+/*eslint-disable*/
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useParams } from 'react-router-dom';
+
+function Descriptions() {
+  const { id } = useParams();
+
+  const [data, setData] = useState([]);
+  useEffect(() => {
+    fetchdetails();
+  }, []);
+
+  const fetchdetails = () => {
+    axios
+      .get(`https://restcountries.com/v3.1/name/${id}`)
+      .then((res) => {
+        setData(res.data);
+        console.log(res.data);
+      })
+      .catch((err) => console.log(err));
+  };
+
+  const country = data.map((d) => {
+    return (
+      //   <img className='card-img-top' src={d.flags.png} alt={d.name.common} />
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.